tests/test_utils.py

```diff
@@ -45,6 +45,10 @@ def test_get_country_code_bad(self):
         country_code = get_country_code(latitude, longitude)
         self.assertEqual(country_code, None)
 
+    def test_get_airports(self):
```

Comment:
> Intuitively, one can view a unit as the smallest testable part of an application. - [Unit testing, WP](https://en.wikipedia.org/wiki/Unit_testing)

This test is neither minimal, nor does it test our code (it mostly tests the seeyou-parser of aerofiles instead).
Imho we should drop it entirely, or at least reduce the test input to a minimal set of entries (1?).